### PR TITLE
Fix formatting in in resolved.conf(5) and nss-resolve(8)

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd-resolved.service.xml [hunk 1]
```diff
@@ -116,28 +118,38 @@
     according to the following rules:</para>
 
     <itemizedlist>
-      <listitem><para>Lookups for the special hostname <literal>localhost</literal> are never routed to the
-      network. (A few other, special domains are handled the same way.)</para></listitem>
+      <listitem><para>Names for which synethetic records as generated (as listed in the previous section) are
```
Comment (mrc0mmand) [inline man/systemd-resolved.service.xml]: *synethetic* -> *synthetic*
*as* -> *are*

File: man/systemd-resolved.service.xml [hunk 2]
```diff
@@ -116,28 +118,38 @@
...
       the LLMNR protocol. Lookups for IPv4 addresses are only sent via LLMNR on IPv4, and lookups for IPv6
       addresses are only sent via LLMNR on IPv6. Lookups for the locally configured host name and the
       <literal>_gateway</literal> host name are never routed to LLMNR.</para></listitem>
 
       <listitem><para>Multi-label names with the domain suffix <literal>.local</literal> are routed to all
-      local interfaces capable of IP multicasting, using the MulticastDNS protocol. As with LLMNR IPv4
+      local interfaces capable of IP multicasting, using the MulticastDNS protocol. As with LLMNR, IPv4
       address lookups are sent via IPv4 and IPv6 address lookups are sent via IPv6.</para></listitem>
 
+      <listitem><para>Single-label names are routed to any interfaces which define search domains, suffixed
```
Comment (dnicolodi) [inline man/systemd-resolved.service.xml]: This sentence is slightly broken. One way to fix it is `to any interfaces` -> `to any interface` and `which define` -> `which defines`, but there are other solutions :-)

File: man/systemd-resolved.service.xml (hunk 1 above)
Comment (keszybz) [reply man/systemd-resolved.service.xml]: Fixed, thanks.

File: man/systemd-resolved.service.xml (hunk 2 above)
Comment (keszybz) [reply man/systemd-resolved.service.xml]: OK, I completely rewrote the paragraph. It's now full of repetitions, but I don't see a way to remove that without losing meaning.

**Comment by keszybz** [comment]
I now removed the commit that described the resolution of single-label names. That description was wrong, and the behaviour is different. I'll work on it, but in the meantime, let's apply the formatting fixes.

**Comment by keszybz** [comment]
Since this is only about formatting and rather trivial, I'll merge this without further review.

**Pull request merged**